src/common/tlv.hpp

```diff
@@ -171,6 +174,24 @@ class Tlv
      */
     Tlv *GetNext(void) { return reinterpret_cast<Tlv *>(static_cast<uint8_t *>(GetValue()) + GetLength()); }
 
+    void SetUdpSourcePort(uint16_t srcPort)
+    {
+        uint16_t *value = reinterpret_cast<uint16_t *>(GetValue());
+        *value          = srcPort;
+    }
+
+    void SetUdpDestionationPort(uint16_t destPort)
+    {
+        uint16_t *value = reinterpret_cast<uint16_t *>(GetValue());
+        value[1]        = destPort;
+    }
+
+    void SetUdpPayload(const void *payload, size_t len)
+    {
+        uint8_t *value = reinterpret_cast<uint8_t *>(GetValue()) + 2 * sizeof(uint16_t);
```

Comment:
use `static_cast` instead of `reinterpret_cast`?

Reply:
Function for udp proxy, not used in this pull request. Code removed.